### PR TITLE
Image with rollovers

### LINKED ISSUE #15: Images Block
with rollovers

see Some of our clients on home page

### PR DESCRIPTION
**Pull request by badvision**
Fix #15 

Test URLs:
- Before: new feature
- After: https://image-with-rollovers--uhg-surest--hlxsites.hlx.page/

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -383,4 +383,37 @@ main .section.two-col .section-container {
 
 .section.footnotes .default-content-wrapper sup {
   font-size: 60%;
-}
\ No newline at end of file
+}
+
+/* images with rollovers */
+.block.images {
```
Comment (shsteimer) [inline styles/styles.css:389]: sorry if this wasn't clear, but I think we should create an images block for this.

File: styles/styles.css [hunk 2]
```diff
@@ -383,4 +383,37 @@ main .section.two-col .section-container {
...
+  display: flex;
+  flex-flow: row nowrap;
```
Comment (shsteimer) [inline styles/styles.css:391]: on mobile, the images get awfully small. probably do flex direction column and then switch it at table breakpoint?

File: styles/styles.css [hunk 3]
```diff
@@ -383,4 +383,37 @@ main .section.two-col .section-container {
...
+  max-width: 15%;
+}
+
+.block.images > div:hover {
+  filter:grayscale(0%) drop-shadow(0 5px 4px rgb(0 0 0 / 40%));
+  margin:-4px 0 4px;
+  transition:filter 1s, margin 1s;
+}
+
+.block.images .icon {
```
Comment (shsteimer) [inline styles/styles.css:413]: I think we probably want to change this so these are images in the word doc so it's easy for authors to update. this is fine for now and we can revisit when time allows later, but just making a note of it.

**Review by shsteimer — CHANGES_REQUESTED** [review]

File: styles/styles.css (hunk 2 above)
Comment (badvision) [reply styles/styles.css:391]: I thought so too, oddly their website shows small images on desktop, but makes the images bigger on mobile.  I'll adjust it, no problem.

File: styles/styles.css (hunk 3 above)
Comment (badvision) [reply styles/styles.css:413]: Both cases work in this PR.  The second image is a PNG image in the original and the rest are SVG currently.  If we change the word doc this should "just work."

**Pull request merged**